homeassistant/components/pyload/config_flow.py

```diff
@@ -0,0 +1,154 @@
+"""Config flow for pyLoad integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import CookieJar
+from pyloadapi.api import PyLoadAPI
+from pyloadapi.exceptions import CannotConnect, InvalidAuth, ParserError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN, HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DEFAULT_HOST, DEFAULT_NAME, DEFAULT_PORT, DOMAIN, ISSUE_PLACEHOLDER
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, description={"suggested_value": DEFAULT_NAME}): str,
```

Comment:
Name is only allowed with helpers

Reply:
what do you mean with helpers?

Reply:
template, generic_*, threshold, statistics

Reply:
ah, now i get it. Name is also a field in the yaml schema, so i just took it over.

Reply:
You can still use that name to set the entry title, but it should not be stored in entry data nor options

Reply:
yep, that's what i'm doing now xD